examples/analogin/main.c

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) 2020-2021 Arm Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ #define MBED_CONF_MBED_TRACE_ENABLE 1
+ #define MBED_TRACE_MAX_LEVEL TRACE_LEVEL_DEBUG
+ #define TRACE_GROUP "main"
+
+ #include "analogin_api.h"
+ #include "mbed_trace.h"
+ #include "bootstrap/mbed_wait_api.h"
+ #include "serial_api.h"
+
+ #include <stdio.h>
+
+ extern serial_t stdio_uart;
+
+ void serial_print(const char *string)
+ {
+     while (*string != 0) {
+         serial_putc(&stdio_uart, *string++);
+     }
+     serial_putc(&stdio_uart, '\n');
+ }
+
+
+
+ int main()
+ {
+     analogin_t analog_input;
+
+     float stored_value;
+     float val;
```

Comment:
```suggestion
     float input_voltage;
     float val;
```

What does val represent? Use your answer to that question to update the variable name to be clearer. Also you may want to declare them closer to usage.